README.md

```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
+  * Not forging new blocks during their turn within dBFT.
+  * Refusal to include block data that does not support them.
+  * Unreasonable policy voting that demonstrates harmful favoritism.
+* Official announcements should always be signed to protect against forgery.
+* The inflation rate should be balanced with the following factors in mind:
+  * Avoiding an inflation rate that would overly devalue existing Swifts.
+  * Real-world effects that would result from a change in the rate.
+  * The amount of income required to provide the basic neccesities in life.
+* Identity Providers should be [banned](#delegated-node-voting) for the following reasons:
+  * Weak approval process allowing sybil accounts--whether due to malice or negligence.
+  * Failing to provide adequate dispute resolution for transactions.
+  * Not performing their duty to approve new Swift Citizens.
+  * Unwillingness to cooperate with other Identity Providers on the platform.
 * Proposals that are approved for funding should remain relatively liberal when the protocol is in its early stages and should slowly become more strict over time.
-* Identity Providers should shield the user from the knowledge that there is a one week incubation period and the fact that transactions can take a few second to process. From the user perspective it is important that SwiftDemand appears to function as efficiently as a centralized system. In the event where something does go wrong it is the responsibility of the Identity Provider to take the loss to rememedy the situation.
+* Identity Providers should shield the user from the knowledge that there is a one-week incubation period and that transactions can take a few seconds to process. From the user's perspective, it's important that SwiftDemand appears to function as efficiently as a centralized system. In the event something does go wrong, it's the responsibility of the Identity Provider to take the loss to remedy the situation.
 
 ## Connection To SwiftDemand
-SwiftDemand is separate from the Swift Protocol, however they are tightly connected and currently being developed by the same team. SwiftDemand will be the initial Identity Provider on the Swift Protocol and all Swifts that exist on SwiftDemand at the time of launching the Swift Mainnet will be transferred at a 1:1 ratio. Swift Citizens however will be required to undergo stricter Identity Verification requirements at this time to comply with KYC\[7] and AML\[8]. 
+SwiftDemand is separate from the Swift Protocol. They are tightly connected, however, and currently being developed by the same team. SwiftDemand will be the initial Identity Provider on the Swift Protocol and all Swifts that exist on SwiftDemand at the time of launching the Swift Mainnet will be transferred at a 1:1 ratio. Swift Citizens will be required to undergo stricter Identity Verification requirements at this time to comply with KYC<sup>[[8]](#8)</sup> and AML<sup>[[9]](#9)</sup>. 
```

Comment:
The sentence `They are tightly connected, however, and currently` sounds a bit odd. Maybe there can be a better way to phrase this.

Reply:
Probably--or instead of changing the word order, change the comma to a semi-colon so it's not a comma splice.